service_mesh/v2x/ossm-troubleshooting-istio.adoc

```diff
@@ -0,0 +1,74 @@
+[id="ossm-troubleshooting"]
+= Troubleshooting your service mesh
+include::modules/ossm-document-attributes.adoc[]
+:context: troubleshooting-ossm
+
+toc::[]
+
+This section describes how to identify and resolve common problems in {ProductName}. Use the following sections to help troubleshoot and debug problems when deploying {ProductName} on {product-title}.
+
+// The following include statements pull in the module files that comprise the assembly.
+
+include::modules/ossm-understanding-versions.adoc[leveloffset=+1]
+
+== Troubleshooting Operator installation
+
+In addition to the information in this section, be sure to review the following topics:
+
+* xref:../../operators/understanding/olm-what-operators-are.adoc[What are Operators?].
+
+* xref:../../operators/understanding/olm/olm-understanding-olm.adoc[Operator Lifecycle Management concepts].
+
+* xref:../../support/troubleshooting/troubleshooting-operator-issues.adoc[OpenShift Operator troubleshooting section].
+
+* xref:../../support/troubleshooting/troubleshooting-installations.adoc[OpenShift installation troubleshooting section]
+
+include::modules/ossm-validating-operators.adoc[leveloffset=+2]
+
+include::modules/ossm-troubleshooting-operators.adoc[leveloffset=+2]
+
+== Troubleshooting the control plane
+
+The Istio control plane is composed of *Istiod* which consolidates several previous control plane components (Citadel, Galley, Pilot) into a single binary.   Deploying the `ServiceMeshControlPlane` also creates the other components that make up {ProductName} as described in the xref:../../service_mesh/v2x/ossm-architecture.adoc[architecture] topic.
```

Comment:
what is Istiod? A resource, object, etc?

Reply:
It's one of the components of Istio.  I had to fire up a cluster to answer this.   Service Mesh creates an istiod service, pod, deployment, AND replica set.  In this case I had it bold because I was talking about it as a component.